components/UserProfile.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { useState, useEffect } from "react";
+import {
+  View,
+  Text,
+  Image,
+  ScrollView,
+  TouchableOpacity,
+  SafeAreaView,
+} from "react-native";
+import { SHADOWS } from "../theme";
+import { StackNavigationProp } from "@react-navigation/stack";
+import { baseUrl } from "../env";
+import { useNavigation } from "@react-navigation/native";
+import { dummyData } from "../DummyData";
+import axios, { AxiosError } from "axios";
+
+const users = dummyData;
+
+type User = {
```

Comment:
In Call: User Objekt wird aufgespalten in einzelne Attribute als Props.

-> Somit ist Liste wiederverwendbar für: FollowerListe anzeigen, Nutzerliste bei Nutzersuche Anzeigen, evtl. weiter anforderungen.

Ob attribute dann angezeigt werden oder nicht wird per if abfrage gehandled